tequilapi/endpoints/proposals.go

```diff
@@ -0,0 +1,38 @@
+package endpoints
+
+import (
+	"github.com/julienschmidt/httprouter"
+	"github.com/mysterium/node/server"
+	dto_discovery "github.com/mysterium/node/service_discovery/dto"
+	"github.com/mysterium/node/tequilapi/utils"
+	"net/http"
+)
+
+type proposalsEndpoint struct {
+	mysteriumClient server.Client
+}
+
+type proposalsDto struct {
+	Proposals []dto_discovery.ServiceProposal `json:"proposals"`
+}
+
+func NewProposalsEndpoint(mc server.Client) *proposalsEndpoint {
+	return &proposalsEndpoint{mc}
+}
+
+func (pe *proposalsEndpoint) List(resp http.ResponseWriter, req *http.Request, params httprouter.Params) {
+
+	nodeId := req.URL.Query().Get("nodeid")
+	proposals, err := pe.mysteriumClient.FindProposals(nodeId)
+	if err != nil {
+		utils.SendError(resp, err, http.StatusInternalServerError)
+		return
+	}
+	proposalsDto := proposalsDto{proposals}
```

Comment:
I don't think it's a good idea to pass json structure as is from discovery service to tequila endpoint response - any change inside discovery service response will automatically break tequila api consumers. What about creating our own ProposalDto and putting only required data? For example node id, proposal number ant some human readable/ decision important info like price etc.

Reply:
done